common/shlibs

```diff
@@ -4196,7 +4196,8 @@ libsword-1.8.1.so libsword-1.8.1_6
 libgivaro.so.9 givaro-4.1.1_1
 liblinbox.so.0 linbox-1.6.3_1
 libpari-gmp-tls.so.8 pari-2.15.0_1
-libtree-sitter.so.0 tree-sitter-0.19.0_1
+libtree-sitter.so.0 tree-sitter-0.24.3_1
+libtree-sitter.so.0.24 tree-sitter-0.24.3_1
```

Comment:
the full soname shouldn't be necessary

Reply:
While building `xbps-src` complained that the *.24 SONAME should also be part of `common/shlib`.

But I can check again.

Reply:
I've tested again and building neovim fails with the following shlibs error:

```
=> neovim-0.10.2_2: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libluv.so.1 <-> libluv>=1.30.1.0_1
   SONAME: libvterm.so.0 <-> libvterm>=0.3_1
   SONAME: /usr/lib/lua/5.1/lpeg.so <-> lua51-lpeg>=1.1.0_2
   SONAME: libmsgpack-c.so.2 <-> msgpack>=1.1.0_1
   SONAME: libtree-sitter.so.0.24 <-> UNKNOWN PKG PLEASE FIX!
   SONAME: libunibilium.so.4 <-> unibilium>=2.0.0_1
   SONAME: libluajit-5.1.so.2 <-> LuaJIT>=2.0.0_1
   SONAME: libm.so.6 <-> glibc>=2.39_1
   SONAME: libuv.so.1 <-> libuv>=1.0.0_1
   SONAME: libc.so.6 <-> glibc>=2.39_1
=> ERROR: neovim-0.10.2_2: cannot guess required shlibs, aborting!
```

Reply:
With amazing help from the IRC I've changed the common/shlib to use the correct value.